themes/helm/layouts/partials/sidebar.html

```diff
@@ -25,19 +25,19 @@ <h1>
         </li>
 
         {{ range $docsSections }}
```

Comment:
The range over the `$docsSections` can be dropped. `.Pages` is a not part of it. I think this can be entirely removed from the document.